modules/manage/pages/manage-security/configure-client-certificates.adoc

```diff
@@ -7,26 +7,106 @@ certificates.
-page explains how client certificates can be prepared for a Java client. It
-also provides information on TLS levels, and on supported ciphers.
+Couchbase clients can authenticate by means of X.509 certificates.
+This page provides step-by-step instructions for the creation of client certificates for:
 
-For a list of Couchbase-Server ports that provide secure connectivity to
-clients, see
+* _Couchbase Server_.
+The certificate can be used by a Couchbase Server-cluster that wishes to secure its connection to another Couchbase Server-cluster.
+This certificate might be used by a _source_ cluster that wishes to perform _Cross Data Center Replication_ securely, to a _destination_ cluster.
+
+* _Java Applications_.
+A Java application based on the Couchbase SDK can obtain its client certificate from a Java _keystore_, and so authenticate with Couchbase Server securely.
+
+For a list of Couchbase-Server ports that provide secure connectivity to clients, see
 xref:learn:clusters-and-availability/connectivity.adoc[Connectivity].
 
-[#cert_auth_for_java_client]
-== Configure Certificates for a Java Client
+[#cert-auth-for-couchbase-server]
+== Configure a Client Certificate for Couchbase Server
+
+The instructions below create a client certificate whose chain of authority is the same as the node certificate created in xref:manage:manage-security/configure-server-certificates.adoc[Configure Server Certificates].
+The instructions assume that _that_ procedure has already been followed; and duly make use of the resulting directory structure and files.
+
+The instructions also assume that the instance of Couchbase Server to be accessed by the client:
 
-Once the root certificate for a Couchbase Server-node has been deployed, a
-Java client can authenticate by means of an appropriately prepared keystore,
-and so gain access to the Data, Query, and Search Services.
+* Contains an instance of the sample bucket `travel-sample`: this is the bucket whose contents the client wishes to read and write.
+For information on sample buckets and how to install them, see xref:manage:manage-settings/install-sample-buckets.adoc[Sample Buckets].
+
+* Has a defined, locally authenticated user named `clientuser`, who has been assigned a role that permits reading and writing to the `travel-sample` bucket.
+For information on creating users and roles, see xref:manage:manage-security/manage-users-and-roles.adoc[Manage Users and Roles].
 
 Proceed as follows.
-Note that these instructions use the Ubuntu 16 environment previously configured
-in
-xref:manage:manage-security/configure-server-certificates.adoc[Configure
-Server Certificates].
+Note that additional information on file-types can be found in the procedure for _server_-certificate generation; in xref:manage:manage-security/configure-server-certificates.adoc[Configure Server Certificates].
+
+. Create and access a working directory.
++
+----
+cd certfiles
+mkdir client
+cd client
```

Comment:
```suggestion
```

Instruction to cd in `client` should be moved to step 3 I think. At least that's the order I followed.